packages/gestalt-datepicker/src/DatePickerTextField.js

```diff
@@ -67,6 +67,7 @@ function DatePickerTextField(props: Props) {
             }}
             errorMessage={errorMessage}
             helperText={helperText}
+            inputMode="none"
```

Comment:
This is a breaking change that we should pull out to a separate PR

Reply:
Breaking change or more like a bug fix?

Reply:
DONE

Reply:
a bug fix can be a breaking change 🙂